exporter/exporter.go

```diff
@@ -169,9 +169,9 @@ func Run(args ...string) error {
 	}
 
 	// default to public API environment
-	lightstepEnv := "public"
-	if len(os.Getenv("LIGHTSTEP_ENV")) > 0 {
-		lightstepEnv = os.Getenv("LIGHTSTEP_ENV")
+	lightstepUrl := "https://api.lightstep.com"
+	if len(os.Getenv("LIGHTSTEP_API_BASE_URL")) > 0 {
```

Comment:
I suppose we'll need to alter this to support `LIGHTSTEP_API_URL`, too

Reply:
ah good catch